app/controllers/questions/consent_controller.rb

```diff
@@ -14,29 +14,31 @@ def form_params
     def after_update_success
       GenerateRequiredConsentPdfJob.perform_later(current_intake)
 
-      # client has not yet been routed
-      unless current_intake.client.routing_method.present?
+      # client has not yet been routed, or was previousl determined to have been at capacity
```

Comment:
*previously

Reply:
my y key intermittantly stops working, but worked every time here in this ephemeral comment where it wouldnt matter if it was broken. lol